### ISSUE TITLE
Rails 6.1 'connection_config is deprecated'

### ISSUE THREAD

**Issue by wlipa** [sferik/rails_admin#3339]
Rails 6.1 deprecation warning from rails_admin:

    DEPRECATION WARNING: connection_config is deprecated and will be removed from Rails 6.2
    (Use connection_db_config instead)
    (called from encoding at /home/user/.gem/ruby/2.7.0/gems/rails_admin-2.0.2/lib/rails_admin/adapters/active_record.rb:74)

**Comment by sharvy** [comment]
Looks like it's been taken care of.

```
adapter =
          if ::ActiveRecord::Base.respond_to?(:connection_db_config)
            ::ActiveRecord::Base.connection_db_config.configuration_hash[:adapter]
          else
            ::ActiveRecord::Base.connection_config[:adapter]
          end
```

**Comment by mshibuya** [comment]
Fixed in 2.1.0.
